### PR TITLE
feat(modal): add calltoAction modal

### PR DESCRIPTION
**Pull request by davidchan1219**
**What this PR does** 📖

Add callToAction modal

**Which issue(s) this PR fixes** 🔨

AP-1059

### CODE REVIEW

**Comment by phillsatellite** [comment]
Tested: Works well but I noticed I was able to replicate bug still if I uploaded image, canceled it/click out of text box, and then re upload again pressing enter to send will bring up file browser

https://user-images.githubusercontent.com/93608357/157913638-fc18cf5f-e709-4dbe-8de2-e30b05ddd028.mov

**Comment by stavares843** [comment]
Captura de ecrã 2022-03-11, às 17 23 08

this PR

vs figma

Captura de ecrã 2022-03-11, às 17 23 19

missing the gradient on the modal background

**Comment by stavares843** [comment]
Captura de ecrã 2022-03-11, às 17 23 08

this PR

vs figma

Captura de ecrã 2022-03-11, às 17 23 19

the keep me posted button should be higher in height and should have a glow around it

**Comment by stavares843** [comment]
Captura de ecrã 2022-03-11, às 17 23 08

this PR

vs figma

Captura de ecrã 2022-03-11, às 17 23 19

the line height between the sentences is not the same

**Comment by stavares843** [comment]
Captura de ecrã 2022-03-11, às 17 23 08

this PR

vs figma

Captura de ecrã 2022-03-11, às 17 23 19

the modal size is not the same as figma

**Comment by stavares843** [comment]
Captura de ecrã 2022-03-11, às 17 23 08

this PR

vs figma

Captura de ecrã 2022-03-11, às 17 23 19

the white line should be light grey to match figma

**Comment by stavares843** [comment]
figma URL
https://www.figma.com/file/92DcdLCbSjqZnoWZ2lznkm/Disabled-Button-Pop-up?node-id=3%3A1609

**Comment by stavares843** [comment]
also please make the following modifications on mobile view as indicated by Liz

image

**Comment by josephmcg** [comment]
@stavares843 should be better. I matched as much as I could within reason. A lot of the more minor inconsistencies will be caught in a global css update post alpha.

In the future, just one screenshot will do. hid comments to make scrolling easier

**Comment by josephmcg** [comment]
confused if this is the best approach. We disabled a lot of the other buttons, but we allow them to click marketplace and show the coming soon modal? Why not just disable the marketplace button?

image

**Review by iltumio — DISMISSED** [review]

**Comment by stavares843** [comment]
because we want the marketplace to add hype, at least that was the decision 

so everywhere where is the marketplace such as the toolbar, glyphs, and so on should display the coming soon modal

File: components/views/glyphs/Glyphs.vue [hunk 1]
```diff
@@ -33,16 +33,17 @@ export default Vue.extend({
...
+        state: !this.ui.modals[ModalWindows.CALLTOACTION],
       })
+      // refactor - AP-1104
+      // const marketInfo = find(marketGlyphs, ({ glyph }) => {
+      //   return glyph.name === this.selectedPack.name
+      // })
+      // this.$store.commit('ui/setGlyphMarketplaceView', {
+      //   view: GlyphMarketViewStatus.SHOP_DETAIL,
+      //   shopId: marketInfo?.id,
+      // })
```
Comment (stavares843) [inline components/views/glyphs/Glyphs.vue:46]: commented-out code 🔨

Comment (josephmcg) [reply components/views/glyphs/Glyphs.vue:46]: yep, saving it for later when functionality is added back

**Comment by stavares843** [comment]
minor thing on the corners
Captura de ecrã 2022-03-18, às 00 28 20
Captura de ecrã 2022-03-18, às 00 28 24

**Comment by stavares843** [comment]
Captura de ecrã 2022-03-18, às 00 33 49
Captura de ecrã 2022-03-18, às 00 33 53

the keep me posted font size seems smaller on figma and the grey line seems more visible on figma as well


besides that is awesome 🎉

**Review by stavares843 — APPROVED** [review]

**Review by WanderingHogan — APPROVED** [review]

**Pull request merged**